sample-applications/blazing-coffee/BlazingCoffee/Client/Pages/Index.razor

```diff
@@ -1,67 +1,146 @@
+                            </Template>
+                        </GridColumn>
+                        <GridColumn Field="@nameof(Employee.IsOnline)" Width="100px" Title="@L["Employee_Grid_Title_Status"]">
+                             <Template>
+                                @{
+                                    var employee = context as Employee;
+
+                                    if (employee != null && employee.IsOnline)
+                                    {
+                                        <span class="k-badge k-badge-md k-badge-solid k-badge-rounded k-badge-success">Online</span>
+                                    }
+                                    else
+                                    {
+                                        <span class="k-badge k-badge-md k-badge-solid k-badge-rounded k-badge-error">Offline</span>
+                                    }
+                                }
+                            </Template>
+                        </GridColumn>
+                    </Columns>
+                </GridColumn>
+                <GridColumn Title="@L["Employee_Grid_Title_Performance"]">
+                    <Columns>
+                        <GridColumn Field="@nameof(Employee.Rating)" Width="110px" Title="@L["Employee_Grid_Title_Rating"]">
+                            <Template>
+                                <Rating CssClass="rating-readonly" Value="((Employee)context).Rating" />
+                            </Template>
+                        </GridColumn>
+                        <GridColumn Field="@nameof(Employee.Target)" Width="230px" Title="@L["Employee_Grid_Title_Engagement"]">
+                            <Template>
+                                @{
+                                    var employee = context as Employee;
+                                    if (employee != null)
+                                    {
+                                       <TelerikProgressBar Max="100" Value="employee.Target" />
+                                    }
+                                }
+                            </Template>
+                        </GridColumn>
+                        <GridColumn Field="@nameof(Employee.Budget)" DisplayFormat="{0:C}" Width="100px" Title="@L["Employee_Grid_Title_Budget"]"></GridColumn>
+                    </Columns>
+                </GridColumn>
+                <GridColumn Title="@L["Employee_Grid_Title_Contacts"]">
+                    <Columns>
+                        <GridColumn Field="@nameof(Employee.Phone)" Width="130px" Title="@L["Employee_Grid_Title_Phone"]"></GridColumn>
+                        <GridColumn Field="@nameof(Employee.Address)" DisplayFormat="{0:C}" Width="200px" Title="@L["Employee_Grid_Title_Address"]"></GridColumn>
+                    </Columns>
+                </GridColumn>
+            </GridColumns>
+        </TelerikGrid>
     }
+</CardContainer>
+
+    @code {
+    private Employee[] employees;
```

Comment:
```suggestion
    private List<Employee> Employees { get; set; } = new List<Employee>();
```
Not sure why not use a list instead. Even if services return arrays, they can be casted with `.ToList()`.